doc/requirements.rst

```diff
@@ -103,6 +107,8 @@ known to be incompatible, or have not been tested together.
 +---------------+-----+-----+-----+-----+
 |               | 2.0 |**N**|  Y  |  Y  |
 +---------------+-----+-----+-----+-----+
+|               | 2.1 |**N**|  Y  |  Y  |
++---------------+-----+-----+-----+-----+
```

Comment:
Let's add a Tornado 6 column, and start testing with Tornado 6 if we haven't already?

Reply:
Done.